hydra/app/models/acda.rb

```diff
@@ -16,7 +16,10 @@ def saved_change_to_preview?
 
   def generate_thumbnail
     # queue job to generate thumbnail
-    GenerateThumbsJob.perform_later(identifier)
+    # we are passing in the Fedora object's id instead of identifier
+    # because the identifier can have special characters which will be
+    # problematic in file and folder creation
+    GenerateThumbsJob.perform_later(id)
```

Comment:
I was expecting the documentation to be added to `GenerateThumbsJob#perform` to explain the parameter.  The receiver (e.g. `#perform` is a good place for that documentation as it conveys that information to LSP help text and via yard).  This inline comment is lost to any supporting development tools (except at this particular location)

Reply:
can do, i'll move it there instead